events/events/urls.py

```diff
@@ -21,5 +21,6 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^$', login_required(TemplateView.as_view(template_name='main.html'), login_url='auth/login?next=/')),
-    url(r'^auth/', include('auth.urls', namespace='auth'))
+    url(r'^auth/', include('auth.urls', namespace='auth')),
+    url(r'^api/v1/events/', include('event.urls', namespace='event'))
```

Comment:
I think we should discuss it later and, if needed, add this changes. By now, everyone follows this structure.